src/io_uring/src/lib.rs

```diff
@@ -150,11 +152,21 @@ impl IoUring {
             len if fd < 0 || (len as i32 - 1) < fd => {
                 Err((Error::InvalidFixedFd(fd), op.user_data()))
             }
-            _ => self.squeue.push(unsafe { op.into_sqe() }).map_err(
-                |err_tuple: (SQueueError, T)| -> (Error, T) {
-                    (Error::SQueue(err_tuple.0), err_tuple.1)
-                },
-            ),
+            _ => {
+                if self.num_ops >= self.cqueue.count() {
```

Comment:
Based on this check I think we now no longer need the check for `IORING_FEAT_NODROP`. We will never have more than `2*size` ops in flight.

Reply:
I would keep it anyway if it doesn't come with any side effects.